docs/spark-qualification-tool.md

```diff
@@ -493,6 +493,82 @@ details on limitations on UDFs and unsupported operators.
 11. _Exec Children Node Ids_
 12. _Exec Should Remove_: whether the Op is removed from the migrated plan.
 
+**Parsing Expressions within each Exec**
+
+The Qualification tool looks at the expressions in the _Exec_ to extract any duration information.
```

Comment:
we don't use duration information for expressions, lets change to something like check to see if the expression is supported by RAPIDS.